tests/viscoplastic_melt_blob_freezing.prm

```diff
@@ -0,0 +1,62 @@
+#########################################################
+# This is a test for the melting rate functionality,
+# in particular the freezing of melt.
+# It uses the melt simple model.
```

Comment:
```suggestion
# It uses the reactive fluid transport material model composited
# on top of the visco plastic material model.
```

Reply:
Hi @naliboff and @jdannberg, I double checked that the only difference in the output between this and the melt simple version of the test is the solid viscosity, so I applied the updated. Thank you!